examples/dood.groovy

```diff
@@ -20,10 +20,10 @@ spec:
 """
   ) {
 
-  def image = "jenkins/jnlp-slave"
+  def image = "jenkins/inbound-agent"
   node(POD_LABEL) {
     stage('Build Docker image') {
-      git 'https://github.com/jenkinsci/docker-jnlp-slave.git'
+      git 'https://github.com/jenkinsci/docker-inbound-slave.git'
```

Comment:
docker-inbound-slave

Reply:
Good catch!

Reply:
Sorry, this was wrong copy-paste, now I fix it.